test/battle_snake/world_test.exs

```diff
@@ -318,11 +322,31 @@ defmodule BattleSnake.WorldTest do
       "game_id" => 0
     }
 
-
     @expected PoisonTesting.cast! @world
 
     test "formats as JSON" do
       assert @expected == @json
     end
+
+    test "consumer formats as JSON with head_url and color" do
+      actual = Poison.decode!(Poison.encode!(@world, consumer: true))
+      expected = @json
+      |> update_in(["snakes"], fn snakes ->
+        snakes
+        |> List.update_at(0, fn snake ->
+          Map.merge(snake,  %{
+                             "head_url" => "snake1.example.com",
+                             "color" => "color1",
+                         })
+        end)
+        |> List.update_at(1, fn snake ->
+          Map.merge(snake,  %{
+                             "head_url" => "snake2.example.com",
+                             "color" => "color2",
+                         })
+        end)
+      end)
```

Comment:
rather than coupling this with a prior definition, it's both easier to maintain and read if the entire contents of the structure were copied and pasted in place with the additions.